org.opent2t.sample.multisensor.superpopular/com.smartthings.sensorpod/js/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "opent2t-translator-com-smartthings-sensorpod",
+    "version": "0.2.0",
+    "description": "SmartThings Sensorpod",
+    "homepage": "http://opentranslatorstothings.org",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/opent2t/translators.git"
+    },
+    "main": "thingTranslator.js",
+    "license": "MIT",
+    "scripts": {
+        "test": "ava tests/test.js",
+        "unittest": "ava tests/unitTests.js"
+    },
+    "ava": {
+        "files": [
+            "tests/*.js"
+        ]
+    },
+    "dependencies": {
+        "opent2t-translator-com-smartthings-hub": "^1.2.2",
```

Comment:
Fixed.

---
In reply to: [108838222](https://github.com/openT2T/translators/pull/220#discussion_r108838222) [](ancestors = 108838222)